migration-from-4x.md

```diff
@@ -323,7 +323,25 @@ editor.ui.registry.addMenuItem('example', {
 
 ### Custom dialogs
 
-Dialogs no longer have `height` or `width` settings. The dialog component now uses CSS3 and a predefined `small`, `medium`, and `large` template to specify the dimensions. See [docs]({{site.baseurl}}/ui-components/dialog/) for more information.
+Dialogs are still opened via the `editor.window.open(spec)` api, however a number of configuration options have changed.
+
+##### Removed settings:
+
+| **Removed setting** | **Description** |
+| ------------------- | --------------- |
+| height | The dialog component now uses CSS3 and a predefined `small`, `medium`, and `large` template to specify the dimensions. |
+| width | The dialog component now uses CSS3 and a predefined `small`, `medium`, and `large` template to specify the dimensions. |
+| bodyType | `bodyType` has been merged into the `body` setting.
+| onpostrender | The dialog configuration instead now includes an `initialData` setting for providing the initial state and an api to fetch or update the data.
+| url | The `iframe` dialog component has been removed, which makes the `url` setting irrelevant.
```

Comment:
Wasn't sure `irrelevant` was the right phrase here. I wondered if instead it should have maybe been `obsolete`?